### ISSUE TITLE
can i set job id when use salt

### ISSUE THREAD

**Issue by SolarisYan** [saltstack/salt#35886]
hello, i want to set my own job id instead of the default jid like '20160830143552395143',is there a parameter to specify this?

Salt Version:
           Salt: 2016.3.2
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: 3.2.2
       dateutil: 2.5.3
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: 1.2.3
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.5 (default, Aug 18 2016, 15:58:25)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.7.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
 
System Versions:
           dist: centos 7.2.1511 Core
        machine: x86_64
        release: 3.10.0-327.22.2.el7.x86_64
         system: Linux
        version: CentOS Linux 7.2.1511 Core

**Comment by cachedout** [comment]
Not from the CLI.

If you use SaltClient directly you can do this, however it's not well-tested so I can't vouch for how well this might work, especially across all returners. Right now, I'd consider the ability to do this experimental at best.

**Comment by SolarisYan** [comment]
Yes, i use SaltClient directly,And i will test it if you need some tests about the ability to do this experimental
